sysutils/gearmand/Portfile

```diff
@@ -1,44 +1,46 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           boost 1.0
 PortGroup           github 1.0
 PortGroup           muniversal 1.0
-PortGroup           boost 1.0
+PortGroup           openssl 1.0
 
-github.setup        gearman gearmand 1.1.18
-revision            7
+github.setup        gearman gearmand 1.1.20
+revision            0
 categories          sysutils net devel
 maintainers         nomaintainer
-platforms           darwin
 license             BSD
 
 description         Framework for distributed computation
 long_description    \
-        Gearman provides a generic application framework  to  farm  out  work \
+        Gearman provides a generic application framework to farm out work \
         to other machines or processes that are better suited to do the work. \
-        It allows you to do work in parallel,  to  load  balance  processing, \
-        and to call functions between languages.  It can be used in a variety \
+        It allows you to do work in parallel, to load balance processing, \
+        and to call functions between languages. It can be used in a variety \
         of applications, from high-availability web sites to the transport of \
         database replication events. In other words, it is the nervous system \
         for how distributed processing communicates.
 
 homepage            http://gearman.org/
```

Comment:
https please

Reply:
@herbygillot It does not support https, unfortunately.